app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -93,10 +94,24 @@
     private int chartZoom = 3;
     private boolean hasZoomedChart = false;
 
-    private NumberFormat sgvFormatter;
-    private boolean mmolxl;
+    public static NumberFormat sgvFormatter;
+    public static boolean mmolxl;
     private boolean mmolxlDecimals;
 
+    public static byte radioChannelInuse = 0;
+    public static long timeLastEHSM = 0;
+    public static long timeLastGoodSGV = 0;
+    public static short pumpBattery = 0;
+    public static int countUnavailableSGV = 0;
+
+    // temporary debug stats for CNL connections
+    public static int dbgCNL_enterControlMode = 0;
```

Comment:
Ok and noted that you wanted to keep these beyond being temporary. Would a new class in utils be good for this?